packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2179,6 +2179,8 @@
   "Not_Imported_Messages_Title": "The following messages were not imported successfully",
   "Notification_Desktop_Default_For": "Show Desktop Notifications For",
   "Notification_Duration": "Notification Duration",
+  "Notification_RequireInteraction": "Close Desktop Notifications after user interaction",
```

Comment:
I think this would be better worded as:  "Require Interaction to Dismiss Desktop Notification"

Otherwise this sounds kind of like when set to false that notification might not close after interaction.

Reply:
@geekgonecrazy Thank you for your input.
I'll gladly follow your suggestion.